pkg/kms/barbican/barbican.go

```diff
@@ -5,11 +5,10 @@ import (
 	"github.com/gophercloud/gophercloud/openstack"
 	"github.com/gophercloud/gophercloud/openstack/keymanager/v1/secrets"
 	openstack_provider "k8s.io/cloud-provider-openstack/pkg/cloudprovider/providers/openstack"
-	"k8s.io/klog"
 )
 
 type BarbicanService interface {
-	GetSecret(cfg Config) ([]byte, error)
+	GetSecret(keyID string) ([]byte, error)
```

Comment:
@adisky I also don't understand the purpose of this interface. Don't you mind if I remove it?